### PR TITLE
File 534 redirect fix

### PR DESCRIPTION
**Pull request by kastoestoramadus**

### CODE REVIEW

**Comment by kastoestoramadus** [comment]
Our current S3 wrapper throws exception. We Could rewrite it to more funtional like. What do you think about it @mcarolan ?

**Comment by kastoestoramadus** [comment]
S3JavaSdkService is already direct place dealing with AWS SDK so we can't wrap more SDK.

File: conf/application.conf [hunk 1]
```diff
@@ -57,6 +57,9 @@ aws {
   s3.bucket.upload.quarantine = file-upload-quarantine
   s3.bucket.upload.quarantine = ${?S3_BUCKET_QUARANTINE}
   s3.bucket.env-subdir = local
+  s3.timeout.connection = 5 seconds
+  s3.timeout.request = 19 seconds
```
Comment (kastoestoramadus) [inline conf/application.conf]: Durring biggest outage I've seen some timeouts like 20s, so I prefer to finish before them.

File: conf/application.conf [hunk 2]
```diff
@@ -57,6 +57,9 @@ aws {
...
+  s3.timeout.socket = 29 seconds
```
Comment (kastoestoramadus) [inline conf/application.conf]: File transfers might work while be very slow at the same time.

File: conf/application.conf [hunk 3]
```diff
@@ -57,6 +57,9 @@ aws {
   s3.bucket.upload.quarantine = file-upload-quarantine
   s3.bucket.upload.quarantine = ${?S3_BUCKET_QUARANTINE}
   s3.bucket.env-subdir = local
+  s3.timeout.connection = 5 seconds
```
Comment (kastoestoramadus) [inline conf/application.conf]: initial handshake with S3 should be always considerably fast

**Comment by YuanZHANG910** [comment]
Good for me to merge

**Comment by mcarolan** [comment]
Looks good, nice one

**Pull request merged**